web-apis/bg/spell-checker.js

```diff
@@ -0,0 +1,29 @@
+const spellchecker = require('spellchecker')
```

Comment:
This include should be moved to `/lib/spell-checker.js` and then exported, so that it is referenced in this file as `spellCheckerLib.spellchecker`